src/components/interfaces/HMI_API.xml

```diff
@@ -4783,6 +4783,27 @@
     </function>
     <function name="DialNumber" messagetype="response">
     </function>
+    <function name="OnResetTimeout" messagetype="notification">
+      <description>
+        HMI must send this notification to SDL for method instance for which timeout needs to be reset
+      </description>
+      <param name="requestID" type="Integer" minvalue="0" maxvalue="65535" mandatory="true">
+        <description>
+          Id between HMI and SDL which SDL used to send the request for method in question, for which timeout needs to be reset.
+        </description>
+      </param>
+      <param name="methodName" type="String" mandatory="true">
+        <description>
+          Name of the function for which timeout needs to be reset
+        </description>
+      </param>
+      <param name="resetPeriod" type="Integer" minvalue="0" maxvalue="1000000" mandatory="false">
```

Comment:
Could you please explain why the `maxvalue` is set to 1000000 for this parameter

Reply:
@ShobhitAd in this case, I believe value has been taken directly from [the proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md#basiccommunication-onresettimeout-would-be-added).